File: leetcode-solutions/0009-palindrome-number/solution.java
class Solution {
    public boolean isPalindrome(int n) {
        int x=n;
        int ans=0;
        while(n!=0){
            ans*=10;
            ans+=n%10;
            n/=10;
        }
        if(ans==x&&ans>=0){
            return true;
        }
        else{
            return false;
        }
    }
}

File: leetcode-solutions/0009-palindrome-number/solution.py
class Solution(object):
    def isPalindrome(self, x):
        n=x
        rev=0
        while n>0:
            digit=n%10
            rev=rev*10+digit
            n//=10
        if rev==x and rev>=0:
            return True
        else:
            return False
